app/controllers/transactions_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TransactionsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  before_action :set_transaction, only: %i[show edit update destroy]
+
+  # GET /transactions or /transactions.json
+  def index
```

Comment:
- [ ] You made a great effort! However, there is one thing that needs to be completed. I can see only the category and total amount but I cannot seed the transactions. In transactions, you need to provide all transactions as well.

<img width="403" alt="Screen Shot 2022-09-09 at 14 36 21" src="https://user-images.githubusercontent.com/69011963/189321083-4c69b0d2-feb6-4275-8e0c-83f8fd80fa4a.png">

> Transactions page
> 
> 
> For a given category, the list of transactions is presented ordered by the most recent
> At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category)
> There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction
> When the user clicks on the "Back" button (<), the user navigates to the home page

Reply:
@Rustamxon7 i had implemented it but i guess the previous reviewer was clear about the changes 
Thanks for the review 
![Screenshot from 2022-09-09 11-55-15](https://user-images.githubusercontent.com/38649067/189324575-bbf20431-ac68-4913-acfc-9bd1d1c039ec.png)